enterprise/frontend/src/embedding-sdk/lib/version-utils.unit.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { isSdkVersionCompatibleWithMetabaseVersion } from "./version-utils";
+
+jest.mock("../config", () => ({
+  getEmbeddingSdkVersion: jest.fn().mockReturnValue("1.0.0"),
+}));
```

Comment:
```suggestion
```

we don't seem to use `getEmbeddingSdkVersion` inside `isSdkVersionCompatibleWithMetabaseVersion`, so there's no need to mock it.